src/Altinn.Profile.Core/OrganizationNotificationAddresses/Organization.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Altinn.Profile.Core.OrganizationNotificationAddresses
+{
+    /// <summary>
+    /// class for organizations connection id and orgNumber
+    /// </summary>
+    public class Organization
```

Comment:
Trenger tips til navngiving. Denne modellen er prikk lik som den i integrations.Entities. Jeg vill ikke dra inn data-annotations inn i core-prosjektet, så gjenbruk av klasse er nok utelukket. Nå har jeg kalt begeg to Organization, men da blir det stadig "ambiguous"-warnings